lib/partition_setup.pm

```diff
@@ -117,13 +118,24 @@ sub create_new_partition_table {
     }
 }
 
+# Set mount point and volume label
 sub mount_device {
     my ($mount) = shift;
-    send_key 'alt-o' if is_storage_ng;
+    send_key 'alt-o';
```

Comment:
Yes,I have tested it on SLE12SP4 and passed. 
If this change has some risk,it's ok for me remain unchanged on this commit,since our next urgent testing is SLE15SP1 bate2.